src/main/java/io/github/skyblockcore/mixin/HandledScreenMixin.java

```diff
@@ -0,0 +1,78 @@
+package io.github.skyblockcore.mixin;
+
+import io.github.skyblockcore.SkyblockCore;
+import io.github.skyblockcore.command.SkyblockCoreCommand;
+import io.github.skyblockcore.event.ConfigManager;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.ingame.HandledScreen;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.ScreenHandler;
+import net.minecraft.screen.slot.Slot;
+import net.minecraft.text.HoverEvent;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import org.jetbrains.annotations.Nullable;
+import org.lwjgl.glfw.GLFW;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import static io.github.skyblockcore.SkyblockCore.TITLE;
+
+@Mixin(HandledScreen.class)
+public class HandledScreenMixin {
+
+    @Shadow
+    @Nullable
+    protected Slot focusedSlot;
+
+    @Shadow
+    @Final
+    protected ScreenHandler handler;
+
+    /**
+     * Listens for when the player presses the key to copy NBT data from an item to clipboard.
+     * Also prints the item copied in chat, with hover-over showing the lore.
+     *
+     * @param keyCode The key code of the key that was pressed
+     * @param scanCode Unused, the physical location of the key on the keyboard
+     * @param modifiers The modifiers that were active when the key was pressed (e.g. shift, ctrl, caps, etc.)
+     * @param cir The callback info
+     */
+    @Inject(method = "keyPressed", at = @At("HEAD"))
+    void onKeyPressed(int keyCode, int scanCode, int modifiers, CallbackInfoReturnable<Boolean> cir) {
+        // Ensure the NBT copying functionality is enabled.
+        if (!SkyblockCoreCommand.NBTCOPYING) return;
+
+        // Check that the player has pressed the right control key (this should be editable in the future).
+        if (GLFW.GLFW_KEY_RIGHT_CONTROL != keyCode) return;
```

Comment:
Is it worth making this key customisable before we merge this?

Reply:
Should be imo